sdk/communication/azure-communication-administration/azure/communication/administration/_phone_number_administration_client.py

```diff
@@ -387,39 +420,69 @@ def list_all_releases(
         )
 
     @distributed_trace
-    def get_search_by_id(
+    def get_reservation_by_id(
         self,
-        search_id,  # type: str
+        reservation_id,  # type: str
         **kwargs  # type: Any
     ):
         # type: (...) -> PhoneNumberSearch
-        """Get search by search id.
+        """Get reservation by reservation id.
 
-        :param search_id: The search id to be searched for.
-        :type search_id: str
+        :param reservation_id: The reservation id to get reservation.
+        :type reservation_id: str
         :rtype: ~azure.communication.administration.PhoneNumberSearch
         """
         return self._phone_number_administration_client.phone_number_administration.get_search_by_id(
-            search_id,
+            search_id=reservation_id,
             **kwargs
         )
 
     @distributed_trace
-    def create_search(
+    def begin_reserve_phone_numbers(
         self,
         **kwargs  # type: Any
     ):
-        # type: (...) -> CreateSearchResponse
-        """Creates a phone number search.
-
-        :keyword azure.communication.administration.CreateSearchOptions body:
-        An optional parameter for defining the search options.
-        The default is None.
-        :rtype: ~azure.communication.administration.CreateSearchResponse
+        # type: (...) -> LROPoller
+        """Begins creating a phone number search to reserve phone numbers.
+        Caller must provide either body, or continuation_token keywords to use the method.
+        If both body and continuation_token are specified, only continuation_token will be used to
```

Comment:
Let's rename `body` something more user friendly kwarg name. The user has to do think hard to understand what is `body`

Reply:
Yes that is a valuable point. We need to sync with other team members to make the args here consistent across languages.

Reply:
I have renamed `body` to `options` when starts a phone number reservation.

I think it is great to flatten the CreateSearchOptions, but let's reach a consensus with other team members, and make it a separate PR after the consensus.